docs/UserGuide.md

```diff
@@ -41,146 +38,415 @@ By : `Team SE-EDU`  &nbsp;&nbsp;&nbsp;&nbsp; Since: `Jun 2016`  &nbsp;&nbsp;&nbs
+
+Format: `find undone KEYWORD [MORE_KEYWORDS]`
+
+> * Similar to regular `find` feature, but only searches through `undone` tasks
+
+### 2.17. Find done tasks: `find done`
+Finds done tasks whose properties contain any of the given keywords.
 
-> Deletes the person at the specified `INDEX`. <br>
-> The index refers to the index number shown in the most recent listing.<br>
+Format: `find done KEYWORD [MORE_KEYWORDS]`
+
+> * Similar to regular `find` feature, but only searches through `done` tasks
+
+### 2.18. Check off task to mark as done: `check`
+Checks off a task to mark as done.
+> Checks off the task at the specified `INDEX`.
+> The index refers to the index number shown in the most recent listing.
 > The index **must be a positive integer** 1, 2, 3, ...
 
 Examples:
 
-* `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br>
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+* `list`
+  `check 2`
+  Checks off the 2nd task in the listing as done.
+* `find Essay`
+  `check 1`
+  Checks off the 1st task in the results of the `find` command.
+
+### 2.19. Uncheck task to mark as undone: `uncheck`
+Uncheck a task to mark as undone.
+> Unchecks the task at the specified `INDEX`.
+> The index refers to the index number shown in the most recent listing.
+> The index **must be a positive integer** 1, 2, 3, ...
+
+Examples:
 
-### 2.7. Select a person : `select`
+* `list`
+  `uncheck 2`
+  Unchecks the 2nd task in the listing marking it as undone.
+* `find Essay`
+  `uncheck 1`
+  Unchecks the 1st task in the results of the `find` command.
 
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+### 2.20. See usage instructions: `help`
+Shows a list of commands that Suru accepts.
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`.<br>
-> The index refers to the index number shown in the most recent listing.<br>
-> The index **must be a positive integer** 1, 2, 3, ...
+### 2.21. See specific command's usage instructions: `help COMMAND`
+Shows usage instractions for a specific command.
 
 Examples:
 
-* `list`<br>
-  `select 2`<br>
-  Selects the 2nd person in the address book.
-* `find Betsy` <br>
-  `select 1`<br>
-  Selects the 1st person in the results of the `find` command.
+* `help add`
+* Shows usage instructions for the `add` command.
 
-### 2.8. Clearing all entries : `clear`
+### 2.22. Define save destination: `saveto`
+Allows user to define destination to save to.
 
-Clears all entries from the address book.<br>
-Format: `clear`
+Examples:
 
-### 2.9. Exiting the program : `exit`
+* `saveto` `FILE_PATH`
 
-Exits the program.<br>
-Format: `exit`
+### 2.23. Define target to load from: `loadfrom`
+Allows user to define target to load from.
+
+Examples:
+
+* `loadfrom` `FILE_PATH`
+
+### 2.24. Add reminders to task: `add DESCRIPTION reminders: ...`
```

Comment:
I guess from here on are your lower priority features?  It would be good to include them in the relevant sections as and when you make time for them.